### PR TITLE
Mqtt client index tests

### PR DESCRIPTION
**Pull request by andrewqiamzn**
### Why?
The DAB-ADB bridge currently has no tests. We want to add unit tests with Jest and achieve minimum 80% coverage.
### DOD
- [x] Achieve 80% test coverage `mqtt_client/index.ts`
### Test Plan
image

### CODE REVIEW

File: adapters/dab-adb-bridge/src/lib/adb/__tests__/adb_commands.test.ts [hunk 1]
```diff
@@ -0,0 +1,969 @@
+import { AdbCommands } from "../adb_commands";
```
Comment (andrewqiamzn) [inline adapters/dab-adb-bridge/src/lib/adb/__tests__/adb_commands.test.ts:1]: Will rebase this branch from [adb commands](https://github.com/device-automation-bus/dab-tools/pull/12) branch once that has been merged and this change should remove itself

**Review by amzn-stenlaan — APPROVED** [review]
